src/main/webapp/app/exercises/modeling/manage/modeling-exercise.service.ts

```diff
@@ -27,11 +27,11 @@ export class ModelingExerciseService {
             .pipe(map((res: EntityResponseType) => this.exerciseService.convertDateFromServer(res)));
     }
 
-    update(modelingExercise: ModelingExercise, req?: any): Observable<EntityResponseType> {
+    update(modelingExercise: ModelingExercise, exerciseId: number, req?: any): Observable<EntityResponseType> {
```

Comment:
Isn't the `exerciseId` argument redundant here since it can be retrieved from `modelingExercise`?

Reply:
agreed

Reply:
Ok, I reverted this change